python/ray/data/_internal/execution/interfaces/op_runtime_metrics.py

```diff
@@ -118,17 +115,16 @@ def extra_metrics(self) -> Dict[str, Any]:
         """Return a dict of extra metrics."""
         return self._extra_metrics
 
-    def as_dict(self, metrics_only: bool = False):
+    def as_dict(self, metrics_only: bool = True):
```

Comment:
I have modified the `as_dict` function here to have `metrics_only` default to `True` and have made it so unless a metric explicitly has `export_metric: False` then the metric will be exported. The same metrics will be exported but this allows us to properly hide the backpressure metrics for now.